tester_app/app/assets/javascripts/application.js

```diff
@@ -10,7 +24,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery
-//= require jquery_ujs
+
+
```

Comment:
@justin808 I'm doing this in sort of hack-y way where I use [gsub_file](http://www.rubydoc.info/github/wycats/thor/master/Thor/Actions#gsub_file-instance_method) to replace the target text with just `""`, I guess I could then also run gsub_file again and look for two blank lines in a row and replace with one? Talking to myself...